qa/L0_java_memory_growth/test.sh

```diff
@@ -38,7 +38,9 @@ set -e
 
 MODEL_REPO=`pwd`/models
 SAMPLES_REPO=`pwd`/javacpp-presets/tritonserver/samples
-cp Simple.java $SAMPLES_REPO
+rm $SAMPLES_REPO/Simple.java
+cp MemoryGrowthTest.java $SAMPLES_REPO/MemoryGrowthTest.java
+sed -i 's/Simple/MemoryGrowthTest/g' $SAMPLES_REPO/pom.xml
```

Comment:
Does it make sense if we add `pom.xml` in this test folder, and it is dedicated to build the memory growth test? Then the pom modification below shouldn't be needed as well.

Reply:
Good suggestion. The pom.xml and Java file are now located under the test subdirectory (to prevent Maven from searching the subdirectories for other build files and getting confused).